test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Feeds/InstalledAndTransitivePackageFeedTests.cs

```diff
@@ -237,7 +237,7 @@ public async Task SearchAsync_WhenMultitargetingAndEmptySearch_ReturnsOnePackage
                 e => Assert.Equal(new PackageIdentity("packageC", new NuGetVersion("2.0.0")), e.Identity),
                 // Returns latest transitive package version
                 e => Assert.Equal(new PackageIdentity("transitivePackageC", new NuGetVersion("0.0.2")), e.Identity),
-                // Returns all the transtive packages that are not the latest verion
+                // Returns all the transitive  packages that are not the latest version
```

Comment:
Remove extra whitespace in comment.

```suggestion
                // Returns all the transitive packages that are not the latest version
```